src/main/scala/example/Task1RepartitioningApproach.scala

```diff
@@ -0,0 +1,85 @@
+package example
+
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import org.apache.spark.sql.functions.col
+
+object Task1RepartitioningApproach {
+  // QUERY: Find spendings (both already paid or not) of different users on products from different categories.
+
+  def transformationTask1WithRepartitioningApproach(
+      spark: SparkSession,
```

Comment:
Isn't better to use spark session as a implicit?

Reply:
Maybe it is. I will try to check and reformat.